src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example3.java

```diff
@@ -0,0 +1,53 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces">
+      <property name="allowSingleLineStatement" value="true"/>
+      <property name="tokens"
+             value="LITERAL_IF, LITERAL_WHILE, LITERAL_DO, LITERAL_FOR"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.needbraces;
+
+// xdoc section -- start
+class Example3 {
+  String obj = new String();
+  String value = new String();
+  int counter = 1;
+  int count = 0;
+  int num = 12;
+  String o = "O";
+  public boolean Example3() {
+    if (obj.equals(13)) return true;
+    // ok, because IF is not a target of validation
```

Comment:
Please put real reason, in config IF is a target.

Reply:
Done !